Tests/NetworkingTests/EndpointRequestStorageProcessorTests.swift

```diff
@@ -116,19 +130,17 @@ final class EndpointRequestStorageProcessorTests: XCTestCase {
         }
         
         let model = try JSONDecoder().decode(EndpointRequestStorageModel.self, from: data)
-        
-        XCTAssert(
-            model.statusCode == 200 &&
-            model.method == "GET" &&
-            model.path == mockEndpointRequest.endpoint.path &&
-            model.parameters == ["query": "mock"] &&
-            model.requestBody == nil &&
-            model.requestBodyString == nil &&
-            model.requestHeaders == mockURLRequest.allHTTPHeaderFields &&
-            model.responseBody == mockResponseData &&
-            model.responseBodyString == String(data: mockResponseData, encoding: .utf8) &&
-            model.responseHeaders == ["mockResponseHeader": "mock"]
-        )
```

Comment:
@matejmolnar please don't use something like this. It makes it very hard to understand what is wrong when the test fails.

Reply:
Good point 👍, thanks for the feedback.